packages/terra-status-view/src/StatusView.jsx

```diff
@@ -117,8 +117,8 @@ class StatusView extends React.Component {
       componentsHeight += this.dividerNode.offsetHeight;
     }
 
-    if (this.glyphNode && viewHeight >= componentsHeight + this.glyphNode.offsetHeight) {
-      // if have a glyph to show and can fit inside the containers height, distribute remaining padding to center
+    if (this.glyphNode && viewHeight >= componentsHeight + this.glyphNode.offsetHeight && this.innerNode.offsetWidth >= this.glyphNode.offsetWidth) {
+      // if have a glyph to show and can fit inside the containers height and width, distribute remaining padding to center
```

Comment:
nit: can we reword the first part of this comment, maybe "if glyph exists and can..."